test/skipped/explicit_ret.jl

```diff
@@ -11,7 +11,7 @@ end
 
 mf = test_ex_rt()
 
-for alg = [HMC(2000, 0.2, 3), PG(20, 2000), SMC(10000), IS(10000), Gibbs(2000, PG(20, 1, :x), HMC(1, 0.2, 3, :y))]
+for alg = [HMC(2000, 0.2, 3), PG(20, 2000), SMC(), IS(10000), Gibbs(2000, PG(20, 1, :x), HMC(1, 0.2, 3, :y))]
```

Comment:
What do we do this change only? Should we update all + the `sample` function?

Reply:
That was a find and replace call. I ignored all the skipped tests, and that's why none of the others are changed. Do we have a plan for the skipped tests? @yebai

Reply:
I see. I'm fine with this choice :)